extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -823,6 +837,22 @@ export class Carousel {
       return false;
     }
 
+    return this.forwards_ ? this.isScrollAtStart() : this.isScrollAtEnd();
+  }
+
+  /**
+   *
```

Comment:
nit: add descriptive comment.

This happens to me so often when I just want the red lines to go away 😂

Reply:
Oops! Added comments and renamed these methods.